org.mixedrealitytoolkit.standardassets/Fonts/Fluent Icon Set.asset

```diff
@@ -0,0 +1,302 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
Maybe a better name would align with the font name:  for example, if this asset is using regular, then the file name should be:

"Selawik-Regular-MRTKIcons.asset"

Reply:
Agreed, although since the underlying Icon Font Asset is Selawik-Semibold-MRTKIcons SDF then the name should be "Selawik-Semibold-MRTKIcons.asset".  Do you concur?
I renamed the file in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/700/commits/e1824a1e3f02a795ff60c9092d6f9b29e054905a